lib/OpenQA/Scheduler/Scheduler.pm

```diff
@@ -379,18 +379,10 @@ and a boolean that makes bypass constraints checks about rescheduling.
 
 sub _reschedule {
```

Comment:
Well, i guess this is now not needed at all, as the tick is at fixed value

Reply:
no, it's set to 1ms on wakeup calls

Reply:
wasn't the goal to reduce complexity? ( as,  do we need wakeup still ? )

Reply:
I don't consider that part complex - the wakeup call is there so that if a user clicks on job restart on an idle instance, the job is started then and not 20 seconds later

Reply:
the wakeup call only sets a variable - which is used in schedule to define the next tick. So there is always a tick. The wakeup call only makes sure the next tick comes earlier. 

If you have an easier way to shorten the tick on user activity - go for it. For now I consider this good enough